components/EditForm/index.js

```diff
@@ -8,15 +8,12 @@ import StyledInput from "../Layout/FormStyles/StyledInput";
 import StyledFieldset from "../Layout/FormStyles/StyledFieldset";
 import StyledTextarea from "../Layout/FormStyles/StyledTextarea";
 import StyledSelect from "../Layout/FormStyles/StyledSelect";
-import StyledSubmitButton from "../Layout/FormStyles/StyledSubmitButton/inex";
-import StyledCancelButton from "../Layout/FormStyles/StyledCancelButton";
 import StyledErrorMessage from "../Layout/FormStyles/StyledErrorMessage";
 import StyledMaterials from "../Layout/FormStyles/StyledMaterials";
 import StyledInstructions from "../Layout/FormStyles/StyledInstructions";
-import StyledDeleteButton from "../Layout/FormStyles/StyledDeleteButton";
 import Image from "next/image";
-import StyledAddButton from "../Layout/FormStyles/StyledAddButton";
-import WarningSVG from "@/public/assets/warning_FILL1_wght400_GRAD0_opsz24";
+import WarningSVG from "@/public/assets/WarningIcon";
+import StyledButton from "../Layout/Styled Buttons";
```

Comment:
It would be cleaner to use a strict import method. 🧽

Reply:
We are not sure how to update this, as we used the suggested ways of VSC. Therefore we would keep it if this is no big problem. :)